octoprint_mrbeam/templates/wizard/firstrun_end.jinja2

```diff
@@ -1,14 +1,17 @@
 <h3>{{ _('All Done!') }}</h3>
 
-<p>{% trans %}
-    Your Mr Beam II is now all set up and ready to go. Happy
-    lasering!
-{% endtrans %}</p>
+<p>
+    {% trans %}
+        {# Translators: try number 2 #}
```

Comment:
what does this mean: " try number 2 " ?

Reply:
This was for debugging translation notes